src/EditorFeatures/CSharp/CSharpEditorResources.resx

```diff
@@ -126,4 +126,7 @@
   <data name="Properties" xml:space="preserve">
     <value>Properties</value>
   </data>
+  <data name="UseAutoProperty" xml:space="preserve">
+    <value>Use auto property</value>
```

Comment:
Presumably.  But at that point, i'll move the string when i refactor things to have a common shared codebase :)